django/core/cache/backends/memcached.py

```diff
@@ -203,7 +203,7 @@ def touch(self, key, timeout=DEFAULT_TIMEOUT, version=None):
         key = self.make_key(key, version=version)
         if timeout == 0:
             return self._cache.delete(key)
-        return self._cache.touch(key, self.get_backend_timeout(timeout))
+        return bool(self._cache.touch(key, self.get_backend_timeout(timeout)))
```

Comment:
Do we need to call `bool()`? `touch()` returns `True` or `False` according to `pylibmc` docs.

Reply:
We don't, so will remove. It was just for consistency with `BaseMemcachedCache.touch()` so that the only difference was the `if timeout == 0:` branch which is there due to a crash.

( I almost wonder whether it makes sense to have a `delete_on_touch_with_zero_timeout` feature flag for that.)